definitions/scenarios/packages.rb

```diff
@@ -76,11 +76,15 @@ def compose
                                 :force => true, :warn_on_errors => true)
         else
           add_step_with_context(Procedures::Packages::InstallerConfirmation)
-          add_step_with_context(Procedures::Packages::UnlockVersions)
+          if el?
+            add_step_with_context(Procedures::Packages::UnlockVersions)
+          end
```

Comment:
instead of doing the `if el` here, should the relevant steps have a `confine: el?` instead?

Reply:
Instead of this I absolutely forgot that we now have `package_manager.version_locking_supported?` ; while rebasing I choose to that instead of checking for EL here.